chain/network/src/peer.rs

```diff
@@ -257,7 +256,7 @@ impl Peer {
             },
             PeerMessage::StateResponse(shard_id, hash, payload, receipts) => {
                 NetworkClientMessages::StateResponse(shard_id, hash, payload, receipts)
-            }
+            },
```

Comment:
your formatter seems different from mine - I just configured clion to run cargo fmt and also run optimize imports first from clion